src/components/Nav.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import hamburgerOpenMenu from '../images/hamburger-open-menu.svg'
+import brigadeLogo from '../images/cfc_logo_2021.svg'
+import slackLogo from '../images/slack.svg'
+import githubLogo from '../images/github.svg'
+import meetup from '../images/meetup.svg'
+
+const Nav = () => {
+  return (
+    <div>
+      <nav class="navbar" aria-label="main">
+        <button aria-label="open" onClick="toggleMenu()" class="menu-icon">
```

Comment:
let's remove the onClick property for now (we'll add that feature in another ticket)